### ISSUE TITLE
Bookmarks Constaint Violation Error

### ISSUE THREAD

**Issue by Typhonragewind** [airsonic-advanced/airsonic-advanced#945]
## Problem description

This morning, without any update or change to the docker container, I couldn't connect to airsonic. Checking the logs it seems it failed to initialize due to a constraint violation error.

### Steps to reproduce

Not sure how to reproduce this

## System information

 * **Airsonic version**: *Edge Release 11.0.0-SNAPSHOT.20220625052932*
 * **Operating system**: *Docker on Ubuntu 20.04*
 * **Client**: *web*
 * **Language**: *english*

## Additional notes
Logs: https://pastebin.com/vYvbe1QC

**Comment by nemogoc** [comment]
I just encountered the exact same error, also didn't change anything. I thought I had automatic backups on, but the only things in my /var/airsonic/backups are from 6 months ago. Oof. Typhon, did you have daily backups or am I looking in the wrong place?

**Comment by Typhonragewind** [comment]
@nemogoc Yes, fortunately my daily backups saved my skin (again). But that is a nice debugging
